### PR TITLE
Generate release post

### PR DESCRIPTION
**Pull request by Grummfy**
Here is an help scripts that generate a post for release with some stats extracted from the github api ;)

### CODE REVIEW

File: scripts/release/build.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+// https://github.com/milo/github-api/wiki/OAuth-token-obtaining
+// https://github.com/milo/github-api/wiki/Pagination
+// https://developer.github.com/v3/repos/commits/
+
+use Milo\Github;
+
+// greating
+echo 'Great, if you are here is to publish a new release, so the team have made a great work!', PHP_EOL;
+echo 'So, just aswer to some questions', PHP_EOL;
```
Comment (vonglasow) [inline scripts/release/build.php]: answer

File: scripts/release/build.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+// taking data
+$currentRevision = readline('Last revision: ');
+$newRevision = readline('New revision: ');
+$author = readline('Your pseudo: ');
+$publicationDate = readline('Publication date for the news(YYYY-MM-DD): ');
+
+$file = __DIR__ . '/../../_posts/' . $publicationDate . '-release-' . str_replace('.', '-', $newRevision) . '.markdown';
```
Comment (vonglasow) [inline scripts/release/build.php]: use `dirname(__DIR__, 2)`

File: scripts/release/build.php [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		$authors[] = $commit->commiter->login;
+	}
+}
+
+$authors = array_unique($authors);
+$contributors = count($authors);
+$fileChanged = count($results->files);
+
+ob_start();
+include __DIR__ . '/release.markdow.tpl.php';
```
Comment (vonglasow) [inline scripts/release/build.php]: it's not `markdown` ?

**Review by vonglasow — CHANGES_REQUESTED** [review]
Small changes but looks cool. Doesn't try to make it works yet

File: scripts/release/build.php (hunk 2 above)
Comment (Grummfy) [reply scripts/release/build.php]: connaissait pas \o/

File: scripts/release/build.php (hunk 3 above)
Comment (Grummfy) [reply scripts/release/build.php]: yes

**Comment by Hywan** [comment]
Didn't try but I like the idea.

**Comment by Grummfy** [comment]
I try for the 3.1.0 ;)

**Pull request merged**